src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/helper/EntityDataLoader.kt

```diff
@@ -231,3 +255,10 @@ class DeleteEntityHelper(
     prisonRepository.flush()
   }
 }
+
+class AllowedPrisonHierarchy(
+  val level1: String,
```

Comment:
called these the same name as in implementation code
levelOneCode
levelTwoCode
levelThreeCode
levelFourCode